chart/compass/charts/director/templates/external-certificate-rotation-job.yaml

```diff
@@ -0,0 +1,214 @@
+              - name: CERT_SUBJECT_PATTERN
+                value: {{ printf .Values.global.externalCertConfiguration.subjectPattern .Values.global.externalCertConfiguration.locality .Values.global.externalCertConfiguration.commonName }}
+              - name: EXPECTED_ISSUER_LOCALITY
+                value: {{ .Values.global.externalCertConfiguration.issuerLocality }}
+              - name: CERT_VALIDITY
+                value: {{ .Values.global.externalCertConfiguration.rotationCronjob.certValidity | quote }}
+              - name: CLIENT_CERT_RETRY_ATTEMPTS
+                value: {{ .Values.global.externalCertConfiguration.rotationCronjob.clientCertRetryAttempts | quote }}
+              - name: CERT_SVC_CLIENT_ID
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.name }}
+                    key: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.clientIdKey }}
+              - name: CERT_SVC_CLIENT_SECRET
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.name }}
+                    key: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.clientSecretKey }}
+              - name: CERT_SVC_OAUTH_URL
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.name }}
+                    key: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.oauthUrlKey }}
+              - name: CERT_SVC_TOKEN_PATH
+                value: {{ .Values.global.externalCertConfiguration.tokenPath }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $.Chart.Name }}-{{ .Values.global.externalCertConfiguration.rotationCronjob.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ $.Chart.Name }}-{{ .Values.global.externalCertConfiguration.rotationCronjob.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+rules:
+  - apiGroups: ["*"]
+    resources: ["secrets"]
+    verbs: ["get", "create", "update", "patch"]
```

Comment:
We need the get verb because without it the job fails - 
<img width="1968" alt="image" src="https://user-images.githubusercontent.com/31803034/143431227-360411f4-957a-476d-a912-1dc385009b1d.png">